### DISCUSSION TITLE
No output when using VTKs legacy reader

### DISCUSSION THREAD

**Discussion by Gephi-2017** [pyvista/pyvista#3299] [Q&A]
Hi Im trying to use pyvista's pyvista.read('example.vtk') on a vtk file that consists of graph info (points, vertices, edges and other vertex adn edge data)
However, I get the following error when trying to read it.

Any help or suggestion on reading the vtk file will be helpful

Error:
raise RuntimeError('No output when using VTKs legacy reader')
RuntimeError: No output when using VTKs legacy reader

Thank you

**Comment by banesullivan** [comment]
It's really tough for us to debug what's going on without the actual file. Would you be able to share the problematic file?

**Reply by MatthewFlamm** [reply]
It would be useful to have the results of `pyvista.Report()`.  This particular error message no longer exists in the current code base or the most recent release, or I can't find it.  It would also be useful to have file and line information for the error from the traceback.

**Reply by banesullivan** [reply]
Good catch, @MatthewFlamm!

**Reply by Gephi-2017** [reply]
[pyvistaError.zip](https://github.com/pyvista/pyvista/files/9554753/pyvistaError.zip)

Thank you for the response.
please find the error message and an example of the file im trying to read
